spec/features/system/admin/instance_management_spec.rb

```diff
@@ -48,8 +48,10 @@
       end
 
       scenario 'I can see all instances' do
-        instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        create_list(:instance, 2)
+        last_page = Instance.page.total_pages
+        instances = Instance.order_by_id.page(last_page)
```

Comment:
except when they are split on two pages... maybe we just check that the last page of instances contains at least one of the instances created?